### PR TITLE
Add support for PHP 7.4

### PR DESCRIPTION
**Pull request by snapshotpl**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ php:
   - 7.1
```
Comment (maennchen) [inline .travis.yml:5]: Could you remove PHP 7.1 in the same turn since it's EOL? (also in the `composer.json`)

Comment (snapshotpl) [reply .travis.yml:5]: Sure. In next PR I want to add support for PHP 8

Comment (maennchen) [reply .travis.yml:5]: I'm completely out of the loop on the state of PHP. Are there expected breaking changes that would affect this project?

Comment (snapshotpl) [reply .travis.yml]: Don't know yet :)

**Review by maennchen — APPROVED** [review]

**Review by maennchen — APPROVED** [review]

**Comment by maennchen** [comment]
@snapshotpl Travis is extremely fast today... Maybe I should replace it with GitHub Actions & Coveralls....

**Comment by maennchen** [comment]
@snapshotpl Would it be ok for you if I do not merge this PR to do what I described in the comment above?

**Comment by snapshotpl** [comment]
GA is great idea, however wait some minutes for Travis, finish and merge this and then migrate to GA in separate PR

**Pull request merged**